pom.xml

```diff
@@ -27,7 +27,7 @@
     </scm>
 
     <properties>
-        <java.version>1.8</java.version>
+        <java.version>11</java.version>
```

Comment:
Do we really need to generate Java 11 bytecode here? The library must work with Java8+, but I don't think we need to force Java 11 as the target runtime environment here